docs/examples/gcpbroker/README.md

```diff
@@ -76,7 +76,7 @@ To show the various types of events you can send, you will make three requests:
    `type:greeting`, run the following in the SSH terminal:
 
    ```sh
-   curl -v "http://broker-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
+   curl -v "http://default-brokercell-ingress.cloud-run-events.svc.cluster.local/cloud-run-events-example/test-broker" \
      -X POST \
      -H "Ce-Id: say-hello" \
      -H "Ce-Specversion: 0.3" \
```

Comment:
Can you change this to `1.0`

Reply:
The event-display didn't support 1.0 last time I checked.